.travis.yml

```diff
@@ -1,23 +1,18 @@
-dist: xenial
-sudo: required
-services:
-  - docker
 language: go
 
-git:
-  depth: 1
-
-install:
-- make tools
+go:
+  - 1.12.x
+  - 1.13.x
+  - 1.14.x
+  - tip
 
 matrix:
   fast_finish: true
   allow_failures:
     - go: tip
   include:
-  - go: "1.12.x"
-    name: "Code Lint"
+  - name: "Code Lint"
```

Comment:
linting will only run once and as such we only need to install linting for this step